torch/_dynamo/eval_frame.py

```diff
@@ -98,7 +99,7 @@ def __init__(self, mod: torch.nn.Module, dynamo_ctx):
 
     def _initialize(self):
         # Do this stuff in constructor to lower overhead slightly
-        if isinstance(self._orig_mod.forward, types.MethodType) and skipfiles.check(
+        if isinstance(self._orig_mod.forward, types.MethodType) and check(
```

Comment:
Suggest changing the name 'check()' to 'should_skip()' (or something similar) for a better understanding.